system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -79,6 +81,7 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int
             DBGLOG("ldap_initialize error %s", ldap_err2string(rc));
             return nullptr;
         }
+
         int reqcert = LDAP_OPT_X_TLS_NEVER;
         ldap_set_option(NULL, LDAP_OPT_X_TLS_REQUIRE_CERT, &reqcert);
```

Comment:
This should also be before the LDAP_INIT call and I just confirmed this **does** make a difference for my LDAP server.

Reply:
ok, makes sense and thats what we had in the tester.
@RussWhitehead can you move these 2 lines above the init ?